apps/vtex-my-subscriptions-3/react/tracking/index.tsx

```diff
@@ -5,11 +5,11 @@ import {
   types,
 } from 'vtex.splunk-monitoring'
 
-import { TOKEN } from './splunkSecrets'
+// import { TOKEN } from './splunkSecrets'
 
 // my-subscriptions index on splunk
 const monitoring = new main.SplunkMonitoring({
-  token: TOKEN,
+  token: null,
```

Comment:
is it expected? would you explain what is going on at this point? it's not clear to me

Reply:
The token was set to `null` because we no longer use Splunk. Nowadays we use Git Secret to store an encrypted token. Setting this token to `null` or using the token is application indifferent. I believe @wender's idea was to use `null` to avoid having to go through the bureaucracy of adding new users to the Git Secret ring to gain access to the token.
Going forward, this will be revised to remove these references to Splunk and add a new logging tool.

Reply:
I  was expecting to not have this call on this PR, that's what triggered my comment. If you are already planning to remove it on the a next PR, I'm ok to move forward.